modules/op-release-notes-1-11.adoc

```diff
@@ -14,12 +14,14 @@ In addition to fixes and stability improvements, the following sections highligh
 
 [NOTE]
 ====
-You can use {pipelines-title} on the {product-title} cluster that runs on ARM hardware. This update includes support for the `ClusterTask` resources where images are available and the Tekton CLI tool on ARM hardware.
+Before upgrading to the {pipelines-title} Operator 1.11, ensure that you have at least installed the {product-title} 4.12.19 or 4.13.1 version on your cluster.
```

Comment:
In this sentence, i do not think the words at least add much to what you want to convey since that is a requirement for this task to be completed. I suggest you remove "at least". I think if we delete the words they do not change the sentence much.

Reply:
The word `at least` has a significance here: it points to the minimal requirement here. we mentioned to have at least OCP 4.12.19 or 4.13.1. So, you can have later versions as well but not prior versions.